lib/index.js

```diff
@@ -307,9 +308,29 @@ If you don't have a key, head over to https://binaris.com to request one`);
   (use "bn create node8 hello" to create a Node.js template function in your CWD)`);
 };
 
+/**
+ * Allow feedback by user of a given account.
+ *
+ * @param {object} options - Command line options.
+ */
+const feedbackHandler = async function feedbackHandler(options) {
+  if (!options.email || options.email.length === 0) {
+    throw new Error('Not a valid email.');
+  }
+  if (!options.message || options.message.length === 0) {
+    throw new Error('Not a valid message.');
+  }
+  const funcData = { email: options.email, message: options.message };
```

Comment:
shouldn't the extra arguments get processed as well? is there an easy way to do it? (not needed for this PR)